src/Workspaces/Core/Portable/CodeStyle/CodeStyleOption.cs

```diff
@@ -85,5 +87,27 @@ public static CodeStyleOption<T> FromXElement(XElement element)
                     throw new ArgumentException(nameof(type));
             }
         }
+
+        bool IEquatable<CodeStyleOption<T>>.Equals(CodeStyleOption<T> other)
```

Comment:
it showed up as public API changes. if we are fine with that, I can make it regular interface implementation.